go-app-ussd_higherhealth_healthcheck.js

```diff
@@ -555,20 +571,17 @@ go.app = (function () {
       if (answers.state_tracing) {
         if (risk === "low") {
           text = $(
-            "You are at low risk of having COVID-19. You will still need to complete this risk " +
-            "assessment daily to monitor your symptoms."
+            "Complete this HealthCheck again in 7 days or sooner if you feel ill or you come into contact with someone infected with COVID-19."
           );
         }
         if (risk === "moderate") {
           text = $(
-            "You should SELF-QUARANTINE for 14 days and do HealthCheck daily to monitor " +
-            "symptoms. Try stay and sleep alone in a room that has a window with good air flow."
+            "We recommend you SELF-QUARANTINE for the next 14 days and do this HealthCheck daily to monitor your symptoms. Stay/sleep alone in a room with good air flow."
```

Comment:
Content in the translation sheet is
```
You should SELF-QUARANTINE for 14 days and do HealthCheck daily to monitor symptoms. Try stay and sleep alone in a room that has a window with good air flow.
```
So not sure why this change is being made.